.changeset/proud-horses-know.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/gatsby-theme-doctocat': minor
+---
+
+Remove news from the menu
```

Comment:
super minor (feel free to ignore), but just in case 'news' doesn't immediately register for folks (like myself).

```suggestion
Remove "What's new" (news) link from the menu
```